corehq/ex-submodules/casexml/apps/case/tests/test_bugs.py

```diff
@@ -24,23 +27,20 @@ def test_generate_xml_with_no_date_modified(self):
             CommCareCase(_id='test').to_xml(version)
 
 
-@override_settings(CASEXML_FORCE_DOMAIN_CHECK=False)
-class CaseBugTest(TestCase, TestFileMixin):
-    """
-    Tests bugs that come up in case processing
-    """
+class CaseBugTestCouch(TestCase, TestFileMixin):
```

Comment:
nit: Saw another one of these that ends in `CouchOnly` I think that may be a better convention and easier to grep for